backend/templates/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Is this to serve the base file?

Reply:
was originally, made redundant by swagger